src/mpid/ch4/netmod/ofi/subconfigure.m4

```diff
@@ -39,7 +39,7 @@ AM_COND_IF([BUILD_CH4_NETMOD_OFI],[
     dnl Use embedded libfabric if we specify to do so or we didn't specify and the source is present
     if test "${with_libfabric}" = "embedded" ; then
         ofi_embedded="yes"
-    elif test -z "${with_libfabric}" ; then
+    elif test -z "${with_libfabric}" && -z ${with_libfabric_lib} && -z ${with_libfabric_include} ; then
```

Comment:
only nit is that one of these is in quotes while the others aren't

Reply:
@wesbland can you just quote the 2 additional ones for consistency?

Reply:
👍 Done